examples/usecases/transformers-next-item-prediction.ipynb

```diff
@@ -80,14 +80,14 @@
   {
```

Comment:
<p>Can we store this to disk again?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/1042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>done</p>